multiple-definitions.hlsl

```diff
@@ -0,0 +1,11 @@
+#pragma pack_matrix(column_major)
+
+#line 21 "tests/bugs/multiple-definitions.slang"
+[numthreads(1, 1, 1)]
+void main()
+{
+
+#line 21
+    return;
+}
+
```

Comment:
Is there a reason for this file to be checked in? It seems like maybe it is being checked in on accident.

If this is really needed by the test case, then I have to wonder why this file isn't under `test/bugs/*` and also why it needs to be checked in as plain HLSL.

Reply:
Checked in by accident. removed.